1.14.5/bullseye/Dockerfile

```diff
@@ -97,5 +88,13 @@ EXPOSE 22555 44555 18332
 
 VOLUME ["/dogecoin/.dogecoin"]
 
+COPY entrypoint.py /usr/local/bin/entrypoint.py
+RUN chmod 500 /usr/local/bin/entrypoint.py
+
+# Dependencies install
+RUN apt update && apt install -y \
+    python3 \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Annoying for development, will reinstall python at each modification of entrypoint. Why would you do that ?

Reply:
Documented in my comment earlier in https://github.com/dogecoin/docker/issues/22#issuecomment-988991126:

> *After thinking about this more, I think it's better when an entrypoint change invalidates the cache of the apt install of python, as a cached version may have a python that does not work with the entrypoint code the same that the latest python from the package maintainers repository. This means that if entrypoint changes, it MUST be built (and tested) with the latest python, because otherwise cached build steps can yield different results than non-cached builds ran at the same point in time.*

So, this is a classic security vs efficiency problem. We can turn it back but then we will have to document explicitly that images should be built with `--no-cache` for production deployments (and for any published images.) 

I am going to have to do some research to see how exactly that impacts deployment because I fear that this would eradicate incremental downloads. So where we would save `<number of local builds per release>` of pressure towards apt repositories, we would add `<number of deployments>` of pressure to Docker Hub. The latter is greater than the former at some point or we'd be developing this mostly for the sake of developing, not usage.

I think if we want the best of both worlds, we have to stop using apt and pin a specific python version, then maintain that. At that point, we control the caching. Cost: sweat and tears. I don't think this repository currently has enough participants to really offer continuity for that, but we can maybe plan for something.

*Note: From my personal experience, docker's caching is great for rapid development, but complicates security of deployment. I have on several occasions done development on a separate `Dockerfile.develop` that just optimizes cache by taking changed files last, and use a distinct Dockerfile for production that optimizes for cache invalidation and security. Also, I have significantly reduced my personal reliance on docker to test code under development in favor of local tools in my development environments, to save surprises later.*

Reply:
Reverted for now. Will do security review for deployment later.